assets/js/components/Spinner.jsx

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
-import { computedIconCssClass } from '@lib/icon';
-
 import { EOS_LOADING_ANIMATED } from 'eos-icons-react';
 
-function Spinner({ centered = false }) {
+function Spinner({
+  centered = false,
+  wrapperClassName = '',
```

Comment:
this can be just `className`

Reply:
Yes and also i don't think we need centered any more :D 
I checked out, it's not used in the codebase or did i miss something?